website/content/docs/enterprise/lts.mdx

```diff
@@ -0,0 +1,192 @@
+1.16    | CY24 Q1 (2024-03-27) | CY25 Q1 (1.19 release) 
+1.15    | CY23 Q3 (2023-09-27) | CY24 Q3 (1.18 release) 
+1.14    | CY23 Q2 (2023-06-21) | CY24 Q2 (1.17 release) 
+1.13    | CY23 Q1 (2023-03-01) | CY24 Q1 (2024-03-27) 
+1.12    | CY22 Q4 (2022-10-12) | CY23 Q3 (2023-09-27) 
+
+**Long-term support** extends support for specific Vault versions for an
+additional year with:
+
+- patches for bugs that may cause outages.
+- patches for critical vulnerabilities and exposures.
+- high-risk security patches.
+
+After the first year, LTS Vault versions move from standard support to long-term
+support for an additional year.
+
+<a id="where" />
+
+## Where do I download the LTS binary?
+
+You do not need to download a separate binary for LTS support. Just make sure to
+select an LTS Vault version (e.g., 1.16, 1.19) when you
+[install](/vault/install) or
+[upgrade](/vault/docs/upgrading) your Vault instance.
+
+<a id="when" />
+
+## When are LTS versions released?
+
+As of Vault Enterprise 1.16, the first release of a calendar year includes
+long-term support.
+
+LTS versions overlap by one year with the previous LTS version entering its
+long-term support window when the new LTS version begins its standard support
+window.
+
+<a id="why" />
+
+## Why should I avoid upgrading from an LTS version to a non-LTS version?
+
+Long-term support is intended for Enterprise customers who cannot upgrade
+regularly to stay within the standard support timeline of one year. The
+goal is to establish a predictable upgrade path with a longer timeline rather
+than extending the lifetime for every Vault version.
+
+Long-term support ensures your Vault version continues to receive critical
+patches for an additional year. If you upgrade to a non-LTS version, you are
+moving your Vault instance to a version that lacks extended support, which means
+your Vault version stops receiving updates once it leaves the standard support
+window:
+
+Version | Expected release | Standard support ends  | Long-term support ends
+------- | ---------------- | ---------------------- | ---------------------
+1.19    | CY25 Q1          | CY26 Q1 (1.22 release) | CY27 Q1 (1.25 release)
+1.18    | CY24 Q3          | CY25 Q3 (1.21 release) | No long-term support
+1.17    | CY24 Q2          | CY25 Q2 (1.20 release) | No long-term support
+1.16    | CY24 Q1          | CY25 Q1 (1.19 release) | CY26 Q1 (1.22 release) 
+
+If a newer version of Vault includes features you want to take advantage of, you
+have two options:
+
+1. Wait for the next available LTS release to maintain long-term support.
+1. Upgrade immediately, then upgrade to an LTS release before the standard
+   support window expires.
+
+<a id="how" />
+
+## How do I upgrade my Vault installation?
+
+You should follow whatever upgrade process you currently use to patch or update
+your Vault installation. However, we recommend allowing additional time as
+upgrading from version LTS to LTS+1 may require transitional upgrades to move
```

Comment:
I don't believe we know that LTS - LTS +1 upgrades will require more time, and by the time we release LTS next we will have tested the upgrade path. I suggest erring on the side of removing the warnings here as we will document it when the next LTS version is available

Reply:
I strongly disagree. Our docs need to reflect the current reality, not the possible future (regardless of how probable that future is).

The use of "may" in the warning is strategic. Until we've actually worked out the process, we cannot guarantee that folks *won't* need to allocate additional time or do transitional upgrades.

I understand that we intend to make it less cumbersome, but the fact remains that updating from LTS to LTS+1 *does* involve jumping multiple Vault versions and, at this moment in time, folks need to be prepared for that.

Even if it's relatively frictionless, we can't promise there won't be times where it isn't a more involved process if only because they need to deal with the potential defaults of features introduced in subsequent versions not being appropriate.